sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -57,24 +56,36 @@ import org.apache.spark.sql.util.SchemaUtils
  * The CatalogTable attributes copied from the source table are storage(inputFormat, outputFormat,
  * serde, compressed, properties), schema, provider, partitionColumnNames, bucketSpec.
  *
+ * Use "CREATE TABLE t1 LIKE t2 STORED AS file_format"
```

Comment:
this is not supported now, let's drop the comment.

Reply:
removed